src/Input/NumberInput/index.ts

```diff
@@ -0,0 +1,5 @@
+import NumberInput from './NumberInput';
+
+export { NumberInput };
+
+export default NumberInput;
```

Comment:
why we need both named export and default export?

Reply:
Yes. I saw it done in other components here in the repo.